setup.py

```diff
@@ -58,6 +58,7 @@ def get_version(rel_path):
         "regex",
         "rich",
         "dm-tree",
+        "tensorflow",
```

Comment:
We actually are deliberately avoiding this for dependency resolution issues with pypi. We have faced problems when tensorflow and tensorflow-text are not updated in lockstep, pypi download a ton of versions of each to find a version match. I think this was the one https://github.com/keras-team/keras-nlp/issues/686

We should not add this, and instead do the replace line I suggested before

```
setup_contents = setup_contents.replace(
     '"tensorflow-text"', '"tensorflow-text-nightly", "tf-nightly"'
 )
```

Who knows why `tensorflow-text` depends on `tensorflow` and `tensorflow-text-nightly` does not depend on `tf-nightly`, but we are basically working around that fact.

Reply:
Done